indico/htdocs/sass/partials/_timelines.scss

```diff
@@ -63,25 +72,36 @@
         position: relative;
 
         .i-box-header {
+            @extend %time-information;
             // To match i-timeline-item-label height
             min-height: $min-height;
+            justify-content: flex-end;
```

Comment:
I don't think we want this. `justify-content: flex-end;` on a flex with row direction will align children to the right.